### PR TITLE
fix: data loss

### LINKED ISSUE #258: Adding Task List Cards: Prone to losing data
**Is your feature request related to a problem? Please describe.**
I am always frustrated when I type in a new task, accidentally click somewhere in the app that is not in the text box or "Add Card" button and I lose all my card data I was just creating. 

Sometimes I must retype my card data many times because I must scroll to the "Add Card" button or I just errantly click somewhere else in the App.

**Describe the solution you'd like**
1. Only close card edit when "Cancel" button is clicked. 
2. Maybe save the card rather than close the card when clicking elsewhere in UI. (Err on side of not losing user input)
3. Create CTRL(CMD)+Enter shortcut to save card. This is a standard convention. Most of the time I lose data because I have to scroll and click to save a card.

**Describe alternatives you've considered**
Ensure "Add Card" button is visible before I start typing.
Slow down. Be very careful when clicking "Add Card" button.

**Additional context**
dangit

### PR DESCRIPTION
**Pull request by sekwah41**
This should fix #258 

Though @roldanjr might want to check if the new behavior is acceptable.

I wanted to make it snap to the bottom only If out of view but that was harder than expected due to the bottom being scrollable and semi transparent it causes it to snap where its not visible
image

So the result is you can no longer close tasks by clicking off them (checked and its the same as apps like todoist) and it will snap the new card to the center of the div 
image

### CODE REVIEW

**Review by roldanjr — APPROVED** [review]
LGTM! If I will be delayed in reviewing some of the PRs, you can merge them @sekwah41 without my approval :) I am barely able to test it. For some reason, electron doesn't work on my machine right now.

**Comment by sekwah41** [comment]
@roldanjr could you give me some info on whats up? i may be able to help you.

**Pull request merged**
